src/lua/zencode_sd_jwt.lua

```diff
@@ -416,6 +416,21 @@ When("use signed selective disclosure '' only with disclosures ''", function(ssd
     ssd.disclosures = disclosure
 end)
 
+IfWhen("verify selective disclosure '' contains disclosures ''", function(ssd_name, lis)
+    local ssd = have(ssd_name)
+    local disclosed_keys = have(lis)
+    for _,k in pairs(disclosed_keys) do
```

Comment:
check that this is an array else pairs will act weird

also accept case in which there is a single one being verified fo be found.

Reply:
added checks and possibility to have a single element